changeNicknames.js

```diff
@@ -17,27 +17,53 @@ function pad2 (string) {
   return (string < 10 ? '0' : '') + string
 }
 
+function truncateDecimals (number, digits) {
+	var multiplier = Math.pow(10, digits),
```

Comment:
I think I should've addressed that, but I think it'd be better to use `const` over `var` if you don't plan on changing the variable.

I'll add a linter and maybe pretty printer next time I work on this.